kuma/wiki/views/edit.py

```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8 -*-
-import json
```

Comment:
This was just catching up with a previous commit it seems. We do have flake8 in place that should show unneeded imports as errors in Travis.